src/views/Login/index.js

```diff
@@ -322,49 +256,7 @@ export default class Login extends Component {
           message={
             <span id="message-id">
               Whoops! It looks like you're using Internet Explorer. Unfortunately, Gordon 360
-              doesn't support IE. Please use a modern browser like Chrome or Firefox. (We don't
-              support Edge yet, but we hope to soon!)
-            </span>
-          }
-          action={[
-            <Button
-              onClick={() => (window.location.href = 'https://www.google.com/chrome/')}
-              style={{ color: 'white' }}
-            >
-              Get Chrome
-            </Button>,
-            <Button
-              onClick={() => (window.location.href = 'https://www.mozilla.org/en-US/firefox/new/')}
-              style={{ color: 'white' }}
-            >
-              Get Firefox
-            </Button>,
-            <IconButton
-              key="close"
-              aria-label="Close"
-              color="inherit"
-              onClick={this.handleCloseSnackbar.bind(this)}
-            >
-              <CloseIcon />
-            </IconButton>,
-          ]}
-        />
-        <Snackbar /* Edge browser popup message */
-          style={{ marginTop: '1rem' }}
-          anchorOrigin={{
-            vertical: 'top',
-            horizontal: 'center',
-          }}
-          open={this.isEdge}
-          onClose={this.handleCloseSnackbar.bind(this)}
-          ContentProps={{
-            'aria-describedby': 'message-id',
-            style: { backgroundColor: gordonColors.primary.cyan },
-          }}
-          message={
-            <span id="message-id">
-              Whoops! It looks like you're using Edge. Unfortunately, Gordon 360 doesn't support
-              Edge yet, but we hope to soon! In the meantime, please try Chrome or Firefox.
+              doesn't support IE. Please use a modern browser like Chrome or Firefox.
```

Comment:
```suggestion
              doesn't support IE. Please use a modern browser like Chrome, Firefox, or Edge.
```

Reply:
I suppose this is acceptable since anyone running IE must be on Windows. The one thing is that we have instructions included for installing Chrome and Firefox, but not Edge. It's installed by default on new Windows 10 systems, but if we do have any users using IE, they won't be on Windows 10.

Reply:
Hmmm. Well, I dont think we should advertise that we support it without putting in instructions for installing it on Edge. I think we should just include it here and write instructions in order to cover all bounds. What do you think?

Reply:
I'm not sure what you mean by "installing it on Edge". I was referring to the fact that this dialogue includes links to install Chrome and Firefox (as an alternative to using IE). Since anyone reading the dialogue will be running Windows (since they're using IE), they should have Edge available (although they might not have it by default since they probably won't be on Windows 10). So I guess we need to add a link to the Edge downloads page, and/or instructions for accessing it if you already have it installed.

That seems like it deserves a separate issue though, and doesn't necessarily fit this pull request. Realistically, I shouldn't have edited the login page as much as I did, but a lot of it was outdated/leftover from now solved bugs.

Reply:
I see what you're saying. Up to you on what to do here.